apps/next-app/src/app/products/page.tsx

```diff
@@ -1,20 +1,113 @@
+      description: 'Chuyên cung cấp các đầu đốt',
+      specifications: 'Công suất: 50-5500kw',
+    },
+    {
+      productId: '102',
+      productName: 'Dầu đốt 2',
+      previewImage:
+        'https://hips.hearstapps.com/hmg-prod/images/dog-puppy-on-garden-royalty-free-image-1586966191.jpg?crop=0.752xw:1.00xh;0.175xw,0&resize=1200:*',
+      description: 'Chuyên cung cấp các đầu đốt',
+      specifications: 'Công suất: 50-5500kw',
+    },
+    {
+      productId: '103',
+      productName: 'Dầu đốt 3',
+      previewImage:
+        'https://hips.hearstapps.com/hmg-prod/images/dog-puppy-on-garden-royalty-free-image-1586966191.jpg?crop=0.752xw:1.00xh;0.175xw,0&resize=1200:*',
+      description: 'Chuyên cung cấp các đầu đốt',
+      specifications: 'Công suất: 50-5500kw',
+    },
+    {
+      productId: '104',
+      productName: 'Dầu đốt 4',
+      previewImage:
+        'https://hips.hearstapps.com/hmg-prod/images/dog-puppy-on-garden-royalty-free-image-1586966191.jpg?crop=0.752xw:1.00xh;0.175xw,0&resize=1200:*',
+      description: 'Chuyên cung cấp các đầu đốt',
+      specifications: 'Công suất: 50-5500kw',
+    },
+    {
+      productId: '105',
+      productName: 'Dầu đốt 4',
+      previewImage:
+        'https://hips.hearstapps.com/hmg-prod/images/dog-puppy-on-garden-royalty-free-image-1586966191.jpg?crop=0.752xw:1.00xh;0.175xw,0&resize=1200:*',
+      description: 'Chuyên cung cấp các đầu đốt',
+      specifications: 'Công suất: 50-5500kw',
+    },
+  ];
+  const paginationPage: PaginationType = {
+    totalPages: parseInt(searchParams.get(totalPageParamName) ?? '1'),
+    currentPage: parseInt(searchParams.get(pageParamName) ?? '1'),
+  };
+
   return (
-    <>
-      <Button
-        onClick={() => {
-          router.push('/');
-        }}
-      >
-        Homepage
-      </Button>
-      <div>This is Product Page</div>
-    </>
+    <div className='bg-[#F5F5F5]'>
+      <div className='max-w-7xl mx-auto w-full '>
+        <div className='grid grid-cols-12'>
+          <div className='col-span-12 md:col-span-3'></div>
+          <div className='col-span-12 md:col-span-9 px-[5px] md:px-0 py-[10px] md:py-[30px] flex items-center font-[Roboto]'>
+            <span className='text-[#0F0F0F] md:text-[#850000] text-base md:text-2xl font-semibold md:font-bold mr-[5px]'>
+              {title.toUpperCase()}
+            </span>
+            <span className='text-[#575757] md:text-[#000000] text-[11px] md:text-base font-normal'>{`(${quantity} sản phẩm)`}</span>
+          </div>
+          <div className='col-span-12 md:col-span-3'></div>
+          <div className='col-span-12 md:col-span-9'>
+            <PagingComponent
+              productItems={productItems}
+              onClickItem={onClickItem}
+              paginationPage={paginationPage}
+              handlePageClick={handlePageClick}
+              classItem={showLayoutList ? 'flex flex-row' : 'flex flex-col'}
```

Comment:
can we move this into the component? <PagingComponent layout="list" /> , a ?

Reply:
We can but didn't neccesarily

Reply:
we do, a. It's better for maintenance in the future

Reply:
discussion late